diff.go

```diff
@@ -110,6 +110,8 @@ type DiffFile struct {
 	// The index (SHA1 hash) of the file. For a changed/new file, it is the new SHA,
 	// and for a deleted file it is the old SHA.
 	Index string
```

Comment:
For a deleted file, is this `Index` still the old SHA? or will it become `000000`?

Reply:
Yes,`Index` will become `000000` for deleted files.

Reply:
OK! Then the docstring needs an update.